d3-scripts/src/d3_scripts/yaml_tools.py

```diff
@@ -69,23 +90,8 @@ def lint_yaml(file_name: str, show_problems=True):
         file_name: The filepath to the YAML claim file
         show_problems: Set to `False` to suppress printing linting problems
     """
-    config = YamlLintConfig(
-        r"""
-        extends: default
-        rules:
-            document-start:
-                # so we don't need to start YAML files with ---
-                present: false
-            line-length:
-                # 80 characters is too small for 1080p/4K monitors
-                max: 120
-            indentation:
-                spaces: consistent
-                indent-sequences: consistent
-    """
-    )
     contents = Path(file_name).read_text()
-    problems = yamllint.linter.run(contents, conf=config, filepath=file_name)
+    problems = yamllint.linter.run(contents, conf=__yaml_lint_config(), filepath=file_name)
```

Comment:
Related to the above
```suggestion
    problems = yamllint.linter.run(contents, conf=_yaml_lint_config, filepath=file_name)
```